ludwig/features/audio_feature.py

```diff
@@ -89,12 +104,12 @@ def preprocessing_defaults():
             "in_memory": True,
             "padding_value": 0,
             "norm": None,
```

Comment:
Hmm, is it possible to use the schema you defined here, or would that cause a circular import? Seems like the parameter values are duplicated.

Reply:
So for preprocessing defaults, that is actually going to be an entirely separate PR after the encoder nesting refactor. I wanted to separate it out into it's own PR since it is a slightly complicated refactor in itself so for the time being we can just let the defaults populate with these methods. But you're absolutely right. The end goal is to have the schema populate the defaults and get rid of these static methods for each feature type.

Reply:
gotcha